src/constants/chainInfo.ts

```diff
@@ -114,7 +113,8 @@ const CHAIN_INFO: ChainInfoMap = {
     infoLink: 'https://info.uniswap.org/#/optimism/',
     label: 'Optimism',
     logoUrl: optimismLogoUrl,
-    circleLogoUrl: optimismCircleLogoUrl,
+    // Optimism perfers same icon for both
+    circleLogoUrl: optimismLogoUrl,
```

Comment:
Considering that your updating the circleLogoUrl, I don't think you need to make anny changes to NetworkFilter.tsx

Reply:
The filter needed to be updated to use logos instead of circleLogos, to match how the global network switcher displays